settings/Core.setting.php

```diff
@@ -239,6 +239,27 @@
     'description' => NULL,
     'help_text' => NULL,
   ],
+  'defaultExternUrl' => [
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'defaultExternUrl',
+    'type' => 'String',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => [
+      'class' => 'crm-select2',
+    ],
+    'default' => 'router',
+    'add' => '5.27',
+    'title' => ts('Extern URL Style'),
```

Comment:
Transitional Extern URL style ...

Reply:
Title tweaked:

<img width="332" alt="Screen Shot 2020-05-14 at 2 34 08 AM" src="https://user-images.githubusercontent.com/1336047/81918544-72f6ef80-958b-11ea-8b49-8ca880ec8004.png">

Reply:
Oh, I just noticed your bit about gitlab. Here's another variation:

<img width="734" alt="Screen Shot 2020-05-14 at 2 45 07 AM" src="https://user-images.githubusercontent.com/1336047/81919691-041a9600-958d-11ea-9698-51641cd4441a.png">

Reply:
@totten @eileenmcnaughton I'm confused -- why would a GitLab issue be necessary?

Reply:
@christianwach the theory is that people would only use this setting if they need to because something weird is happening. Once there are no more something weirds then the setting & support for it can go. Probably in a few months we would remove from the form & later on from the code base

Reply:
@eileenmcnaughton Sounds reasonable to me 👍